internal/lightcone/destruction/nowheretorun/nowheretorun.go

```diff
@@ -0,0 +1,53 @@
+package nowheretorun
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	NowheretoRun key.Modifier = "nowhere-to-run"
+)
+
+// Increases the wearer's ATK by 24%/30%/36%/42%/48%.
+// Whenever the wearer defeats an enemy, they restore HP equal to 12%/15%/18%/21%/24% of their ATK.
+
+func init() {
+	lightcone.Register(key.NowheretoRun, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_DESTRUCTION,
+		Promotions:    promotions,
+	})
+
+	modifier.Register(key.NowheretoRun, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnTriggerDeath: onTriggerDeath,
+		},
+	})
+}
+
+func Create(engine engine.Engine, owner key.TargetID, lc info.LightCone) {
+	amtATKPercent := 0.24 + 0.06*float64(lc.Ascension)
+	amtHeal := 0.12 + 0.06*float64(lc.Ascension)
```

Comment:
fix these. In current state, would be `0.3` and `0.18` at ascension 1 which does not match the description